### PR TITLE
chore: upgrade to unreleased sdk version 0.19

### PR DESCRIPTION
**Pull request by wischli**
## fixes KILTProtocol/ticket#544
This PR should be merged once the SDK has released version 0.19 including the Substrate 2.0 compatibility. It does NOT add `privacy enhancement` as I first want to receive feedback for the setup with the workshop CI tests inside the SDK, see [this PR](https://github.com/KILTprotocol/sdk-js/pull/287). However, I added notes that a `privacy enhancement` workshop does not exist yet, but will be added ASAP.

- updates all code snippets to be compatible with current dev-sdk-js
- minor improvements on content

## How to test:

1. Read and review my wiki entry on ["How to update code examples in the KILT workshop"](https://github.com/KILTprotocol/organizational/wiki/How-to-update-code-examples-in-the-KILT-workshop#2-apply-changes-from-workshop-to-sdk) to understand the design principles
2. Review together with [this SDK PR](https://github.com/KILTprotocol/sdk-js/pull/287). Compare [code snippets from SDK workshop CI](https://github.com/KILTprotocol/sdk-js/pull/287/files?file-filters%5B%5D=.json&file-filters%5B%5D=.ts) with snippets here.

## Checklist:

- [x] I have verified that the code works
- [x] I have verified that the code is easy to understand
  - [ ] If not, I have left a well-balanced amount of inline comments
- [x] I have [left the code in a better state](https://deviq.com/boy-scout-rule/)
- [x] I have documented the changes (where applicable)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ In this tutorial, you will:
 
 ✔ Use the KILT SDK to write onto and read from the KILT blockchain.
 
+❌ Not get in touch with our KILT anonymous credentials. But don't worry: There will be a tutorial for this very soon!
```
Comment (rflechtner) [inline README.md]: AFAIK 'getting in touch' is just 'contact' in the sense of contacting someone. @Dudleyneedham ? Can you help?
I could imagine 'We will not introduce our KILT anonymous credentials just yet. But don't worry: There will be a tutorial for this very soon!'

File: Update.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# How to update code examples in KILT workshop
+
+Please see [here](https://github.com/KILTprotocol/organizational/wiki/How-to-update-code-example-in-the-KILT-workshop#2-apply-changes-from-workshop-to-sdk). To be aware of breaking changes to the workshop code snippets, we execute each snippet in the SDK CI pipeline `test_workshop`.
+
```
Comment (rflechtner) [inline Update.md]: if you're pulling the tests into this repo, the references to the sdk will have to be changed, but most of these instructions will probably stay as they are.

File: ctype.md [hunk 3]
```diff
@@ -66,11 +80,37 @@ Open it and paste the following:
...
+This step is **not part of the tutorial** and should be regarded as additional information on how to create a CType in the first place.
+Normally, you would start with a schema which does not have a defined `$id` yet:
+
+[comment]: <copy and paste 🚧 ctypeFromSchema_example from 2_ctypeFromSchema.ts>
+
+```javascript
+const Kilt = require('@kiltprotocol/sdk-js')
+
+const ctype = Kilt.CType.fromSchema({
+  $id: 'kilt:ctype:0x1',
```
Comment (rflechtner) [inline ctype.md]: don't we auto-generate the id?

File: identity.md [hunk 4]
```diff
@@ -26,20 +27,31 @@ All of the code for this step needs to go into this file.
 
 To generate an identity, two methods from the KILT SDK are needed:
 
-* `generateMnemonic`
-* `buildFromMnemonic` // takes a mnemonic as an input, and outputs an `Identity` instance.  
+- `generateMnemonic`
+- `buildFromMnemonic` // takes a mnemonic as an input, and outputs an `Identity` instance.
+
+In version [0.19.0](https://github.com/KILTprotocol/sdk-js/releases/tag/0.19.0) we added the privacy feature among other things. Unfortunately, this made some calls (like creating an identity) asynchronous. Therefore, you have to wrap your functions inside an `async` function to execute them properly. But don't worry, the tutorial will tell you what to do 😇
```
Comment (rflechtner) [inline identity.md:33]: love it :D

File: identity.md [hunk 5]
```diff
@@ -61,11 +73,35 @@ address: 5CUoo2vAegeaZHPNdxZyuMesR3RWYBKHj4jfVyj4FXzpXPuR
...
+async function identities() {
+  const claimerMnemonic = Kilt.Identity.generateMnemonic()
+  console.log('claimer mnemonic:', claimerMnemonic)
+  const claimer = await Kilt.Identity.buildFromMnemonic(claimerMnemonic)
+  console.log('claimer address:', claimer.getAddress())
+
+  const attesterMnemonic = Kilt.Identity.generateMnemonic()
+  console.log('attester mnemonic:', attesterMnemonic)
+  const attester = await Kilt.Identity.buildFromMnemonic(attesterMnemonic)
+  console.log('attester address:', attester.getAddress())
```
Comment (rflechtner) [inline identity.md:93]: that's a good one

**Review by rflechtner — APPROVED** [review]
LGTM ! You will probably change a few details if you add the tests directly to this repo, but if you want to merge this first as is, I'm happy with that.
Maybe double-check the ctype creation.

File: README.md (hunk 1 above)
Comment (wischli) [reply README.md]: You are right.

File: ctype.md (hunk 3 above)
Comment (wischli) [reply ctype.md]: You are right, it's unnecessary when using `fromSchema`.

**Comment by wischli** [comment]
Merging this onto `develop` for now since the `merge` onto `master` is postponed until we release a new SDK version. Since I also want to add an anonymous credential workshop, I'll use the `develop` branch for that.

**Pull request merged**